### PR TITLE
Keep newlines in description

### PR DESCRIPTION
**Pull request by centic9**
The way word-wrapping is implemented
currently removes newlines from the original
text.

This change keeps them in place and thus
leads to better formatting of longer text in
steps.

### CODE REVIEW

**Review by niekvleeuwen — APPROVED** [review]

**Comment by niekvleeuwen** [comment]
Hi @centic9, thanks for contributing to this repo! Just out of curiosity, how did you find this repo? What did you found useful about it?

**Comment by centic9** [comment]
Hi, 

I was looking for a way to backup the content that I created in Polarsteps and found this repository among a few others on Github when searching for "Polarsteps". This repo seemed to do the job fairly nicely with the resulting PDF.

When trying it I found that some things did not work for me, so I thought I'd just send some of those as PRs, maybe others find this tool also useful.

BTW, you need to "merge" the PRs after reviewing, I cannot merge the PRs to your repository because I do not have write access.

**Comment by niekvleeuwen** [comment]
Cool, good to know that you found the repo useful! Feel free to create issues with feature ideas or bugs. 

I merged the PRs, I didn't realize you do not have rights for that.

**Pull request merged**
